README.md

```diff
@@ -8,7 +8,8 @@ A fun utility bot built by and for the Codecademy Community to seperate fun/util
 
 ## Current Commands
 
-| Command   | Arguments  | Permission | Description           |
-| :-------- | :--------- | :--------- | :-------------------- |
-| `/ping`   | N/A        | Everyone   | Displays bot latency. |
-| `/format` | {language} | Everyone   | Formats a codeblock.  |
+| Command       | Arguments   | Permission | Description                                    |
+| :------------ | :---------- | :--------- | :--------------------------------------------- |
+| `/ping`       | N/A         | Everyone   | Displays bot latency.                          |
+| `/format`     | {language}  | Everyone   | Formats a codeblock.                           |
+| `/helpcenter` | {plaintext} | Everyone   | Displays information for accessing helpcenter. |
```

Comment:
I'd consider changing the description to match what we have on our moderation bot:
```
Provides links to Codecademy's Help Center (either embedded or plaintext).
```